app/controllers/trainer_controller.rb

```diff
@@ -8,17 +8,16 @@ def review_card
 
     if check_result[:state]
       if check_result[:distance] == 0
-        flash[:notice] = 'Вы ввели верный перевод. Продолжайте.'
+        flash[:notice] = t(:correct_translation_notice)
       else
-        flash[:alert] =
-            "Вы ввели перевод c опечаткой.
-            Ваш вариант: #{trainer_params[:user_translation]} /
-            Оригинал: #{@card.original_text} /
-            Перевод: #{@card.translated_text}. Продолжайте."
+        flash[:alert] = t 'translation_from_misprint_alert',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.